app/src/main/java/org/openimis/imispolicies/ClientAndroidInterface.java

```diff
@@ -123,7 +123,7 @@
     private Global global;
     private int Uploaded;
     private HashMap<String, String> controls = new HashMap<>();
-    private final String Path = Environment.getExternalStorageDirectory().getAbsolutePath() + "/IMIS/";
+    private final String Path = Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOCUMENTS) + "/IMIS/";
```

Comment:
The code `Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOCUMENTS) + "/IMIS/"` is used lots o time in the project. To make it version dependent (Policies Demo, Policies Release, ....), please add this code as a method in global and get the information from there. Then IMIS folder should be either last part from applicationId (demoProd, demoRelease, ...) or a build variant parameter. In the rest of the code only the specific folder is added. If we decide to change base folder later, then we will update the global method. Thanks.

Reply:
I moved the `getExternalStoragePublicDirectory` call to the `Global` class and referenced it from there. I also changed the name of the IMIS directory based on build variant